### ISSUE TITLE
[GitLab] ...

### ISSUE THREAD

**Issue by chris-ware** [raycast/extensions#8856]
### Extension

https://www.raycast.com/tonka3000/gitlab

#### Error:
```
The API file appears to be corrupted. You may need to reinstall Raycast.


```

### Steps To Reproduce

Not entirely sure it can be reproduced, but I've tried reinstalling raycast and that hasn't solved it

Essentially, once setting a preference on this extension (prior to reinstall, the Laravel Forge extension also), trying to use any command that used a saved value failed.

**Comment by v0y4g3r** [comment]
Same error with [ChatGPT extension](https://www.raycast.com/abielzulio/chatgpt).

**Comment by macbookandrew** [comment]
Hmm…makes me wonder if something in today’s Raycast release broke prefs storage or retrieval? @pernielsentikaer any idea?

**Comment by sxn** [comment]
hey, thanks for the report. This is an issue with Raycast, not the extension. We're currently addressing it and will deploy a hotfix shortly. Sorry for the hassle 🙏

**Comment by tonka3000** [comment]
@sxn Happy that it wasn't me 😁

**Comment by sxn** [comment]
We've just published a hotfix (`1.60.1`) that should address the issue. You can download it from https://www.raycast.com, or run the `Check For Updates` command. ❤️
